mypy/semanal.py

```diff
@@ -4602,7 +4602,7 @@ def visit_with_stmt(self, s: WithStmt) -> None:
         if s.unanalyzed_type:
             assert isinstance(s.unanalyzed_type, ProperType)
             actual_targets = [t for t in s.target if t is not None]
-            if len(actual_targets) == 0:
+            if not actual_targets:
```

Comment:
would prefer to keep this one as-is, as it retains symmetry with the `elif len(actual_targets) == 1` condition immediately below